lib/context/boolean-context/boolean-context.tsx

```diff
@@ -20,10 +20,13 @@ export const BooleanContextProvider = ({
 }: BooleanContextProviderProps) => {
   const [booleans, setBooleansState] = useState(initialValue);
   const setBooleans = useCallback(
-    (newBooleans: Booleans) => setBooleansState({ ...booleans, ...newBooleans }),
-    [setBooleansState, booleans],
+    (newBooleans: Booleans) =>
+      setBooleansState((current) => ({ ...current, ...newBooleans })),
+    [setBooleansState],
   );
-  const value = useMemo(() => ({ booleans, setBooleans }), [booleans, setBooleans]);
+  const value = useMemo(() => {
+    return { booleans, setBooleans };
+  }, [booleans, setBooleans]);
```

Comment:
Is there a reason this one changed?

Reply:
ah i was doing a console log in there and never put it back